.github/workflows/ios_ci.yml

```
@@ -0,0 +1,93 @@
+      MATCH_REPO_URL: ${{ secrets.MATCH_REPO_URL }}
+      APP_STORE_CONNECT_API_KEY: ${{ secrets.APP_STORE_CONNECT_API_KEY }}
+      APP_STORE_CONNECT_API_KEY_ID: ${{ secrets.APP_STORE_CONNECT_API_KEY_ID }}
+      APP_STORE_CONNECT_ISSUER_ID: ${{ secrets.APP_STORE_CONNECT_ISSUER_ID }}
+
+    steps:
+      - name: 🔄 Checkout source code
+        uses: actions/checkout@v4
+
+      - name: 📝 Generate .env.ci from env
+        working-directory: src/fastlane
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)